src/main/java/cn/nukkit/registry/BlockRegistry.java

```diff
@@ -503,7 +503,7 @@ private void registerVanillaBlocks() {
         this.factoryMap.put(DARK_PRISMARINE_STAIRS, BlockStairsDarkPrismarine::new); //258
 
         this.factoryMap.put(PRISMARINE_BRICKS_STAIRS, BlockStairsPrismarineBricks::new); //259
-
+        this.factoryMap.put(DRIED_KELP_BLOCK,BlockDriedKelp::new); //UNK
```

Comment:
Please change `UNK` to `394`.

The block ID list can be seen here: https://minecraft.gamepedia.com/Bedrock_Edition_data_values#Block_IDs